test/unit/deploy_spec_test.rb

```diff
@@ -249,6 +249,34 @@ class DeploySpecTest < ActiveSupport::TestCase
       assert_equal ['bundle exec foo'], definition.steps
     end
 
+    test "task definitions returns an array of VariableDefinition instances" do
+      @spec.expects(:load_config).returns('tasks' =>
+        {'restart' =>
```

Comment:
The indentation is a bit inconsistent here. Also rubocop is complaining about it.